.github/workflows/security.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build:
     name: Trivy
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Better to update to `ubuntu-latest`.

Reply:
I can change this and the release job to `ubuntu-latest`. I'm pretty sure this was a conscious decision though at the time that we made back when we upgraded to 20.04 (https://github.com/prebid/prebid-server/pull/2358) to have them all on the same version as the docker image. That doesn't really seem necessary though as we really only need the test suite to run on that version right?

Reply:
Discussed offline. All workflows will run on `ubuntu-latest` going forward.